### PR TITLE
Add LenovoX1 target

### PR DESCRIPTION
**Pull request by azbeleva**
Add tag for LenovoX1 target, refactored Browser test

### CODE REVIEW

File: Robot-Framework/resources/ssh_keywords.resource [hunk 1]
```diff
@@ -79,8 +111,9 @@ Verify Systemctl status
 
 Start Chromium
     Log To Console  ${\n}Starting Chromium
-    ${command}=     Set Variable    chromium-browser --enable-features=UseOzonePlatform --ozone-platform=wayland
-    ${command_output}=    Execute Command    nohup ${command} > output.log 2>&1 &
+    Execute Command    sh -c 'cat /etc/xdg/weston/weston.ini | grep chromium | cut -c 6- | sed '\''s/"/\\\"/g'\'' > chromium_launcher'
```
Comment (mikatammi) [inline Robot-Framework/resources/ssh_keywords.resource:114]: I wonder what the shell capabilities here are, could we just instead say: `$(cat /etc/xdg/weston/weston.ini | grep chromium | cut -c 6- | sed '\''s/"/\\\"/g'\'')` so we wouldn't need to write intermediate temporary script file.

Also I was thinking are the single and double quotes really matching right in that sed command?

File: Robot-Framework/resources/ssh_keywords.resource [hunk 2]
```diff
@@ -79,8 +111,9 @@ Verify Systemctl status
...
+    Execute Command    chmod 755 chromium_launcher
+    Execute Command    nohup sudo -u ${LOGIN} ./chromium_launcher > output.log 2>&1 &
```
Comment (mikatammi) [inline Robot-Framework/resources/ssh_keywords.resource:116]: Do we need to use sudo here if we are already logged in as a ghaf user?

**Review by mikatammi — COMMENTED** [review]
Few things pointed out in comments, otherwise looks good :+1:

File: Robot-Framework/resources/ssh_keywords.resource (hunk 1 above)
Comment (azbeleva) [reply Robot-Framework/resources/ssh_keywords.resource]: refactored this part of code using your suggestion

Comment (azbeleva) [reply Robot-Framework/resources/ssh_keywords.resource]: also made a method for parsing App path which I plan to use later for other apps

**Review by mikatammi — APPROVED** [review]
LGTM, I will still test

File: Robot-Framework/resources/ssh_keywords.resource (hunk 1 above)
Comment (mikatammi) [reply Robot-Framework/resources/ssh_keywords.resource]: Good, it's always good to refactor things to be more generic :+1:

**Pull request merged**